css/components/hero.css

```diff
@@ -0,0 +1,65 @@
+.container.no-padding-bottom {
+  padding-bottom: 0;
+}
+
+.row.visible-flex.hero-block-position {
+  align-content: space-around !important;
+  justify-content: center;
+  flex-wrap: wrap;
```

Comment:
6 - 7 eilutes nieko nedaro, pabandyk su inspektu istrint, nieks nesikeicia, 8 eilute atstumia foto nuo sekcijos apacios ant didelio ekrano. siulau sito nenaudot, o ant media tiesiog pakeisti flex-direction:column

Reply:
jei nori šią vieta patestuoti, tai yra ant min ekrano. Skirtumas yra

Reply:
taip yra, bet tas !important vistiek nieko nekeicia, ir kai kuriu klasiu mano nuomone per daug. maza ekrana spresciau taip:
<img width="1440" alt="Screen Shot 2022-04-23 at 11 50 47 AM" src="https://user-images.githubusercontent.com/80914644/164887733-bc42a1da-7a44-468e-bd68-d309c231bbab.png">
28-29 eilutej zinoma netinkamas klases pavadinimas, bet tiesiog norejau ta div su visu tekstiniu content pasiekt nekeiciant html

Reply:
dėl important jo, palikau, bet visa kita tai yra tiesiog kitas sprendimo budas. Palygink su dizainu ka turim pradeda skirtis tada labai. Kodėl as taip padariau, kad kuo labiau atitiktų dizaina. Su šiuo tavo kodu pradeda skirtis... Dizainas: 
![image](https://user-images.githubusercontent.com/64682512/164888811-c833cbbf-1f2c-4c1b-9243-f4c6fff076aa.png)
su tavo kodu:
![image](https://user-images.githubusercontent.com/64682512/164888824-7436fc09-00fe-45ed-901a-aad15e34f6b3.png)
Stengiaus be reikalo nekeisti ko nereikia, bet dariau kad kuo labiau atitiktu dizaina neziurint source

Reply:
su mano kodu rezultatas kaip screenshote, specialiai dariau salia atidariusi, kad matytus kaip gaunas, gal kazko nesiuziurejai ka dariau. tavo kodas gal ir veikia, atrodo graziai dizaine, bet ne visos eilutes funkcionalios, su mano variantu galima butu ismest bent 20 eiluciu kodo, o vaizdas kaip screenshote, o ne kaip tu cia atsiuntei

Reply:
nors siaip tu teisus, dizaine viskas veikia, koks skirtumas kad kelios eilutes nieko nedaro, tai koks skirtumas kiek ten to koko ir kaip jis parasytas